TradeItIosTicketSDK2/TradeItPortfolioBalanceEquityPresenter.swift

```diff
@@ -44,7 +44,7 @@ class TradeItPortfolioBalanceEquityPresenter {
         var dayReturnString = ""
 
         if let dayAbsoluteReturn = self.balance?.dayAbsoluteReturn {
-            dayReturnString = NumberFormatter.formatCurrency(dayAbsoluteReturn, displayVariance: true, currencyCode: balance?.accountBaseCurrency)
```

Comment:
If `currencyCode` is `nil`, `currencySymbol` defaults to the symbol dictated by `locale`

Reply:
I don't think we want to default to `locale` though - if I'm set to the Thai locale and my balance returns in AUD then I want to see AUD

Reply:
Whoops, this comment was intended for the dif on `TradeItPortfolioAccountsTableViewManager.swift` `L110`